tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php

```diff
@@ -1519,6 +1519,25 @@ public function doFoo(?Bar $bar): ?Baz {}
 }',
         ];
 
+        yield 'same ?type declaration reversed' => [
+            '<?php
+class Foo {
+    /**
+     *
+     */
+    public function doFoo(?Bar $bar): ?Baz {}
+}',
+            '<?php
+class Foo {
+    /**
+     * @param ?Bar $bar
+     *
+     * @return ?Baz
+     */
+    public function doFoo(?Bar $bar): ?Baz {}
```

Comment:
We probably need also a test case with:

```
     * @param ?Bar $bar
     *
     * @return ?Baz
     */
    public function doFoo(Bar|null $bar): ?Baz {}
```

Reply:
I think they were never added for 8.0 support